runtime/str.go

```diff
@@ -157,6 +157,29 @@ func strAdd(f *Frame, v, w *Object) (*Object, *BaseException) {
 	return NewStr(stringV + stringW).ToObject(), nil
 }
 
+func strCapitalize(f *Frame, args Args, kwargs KWArgs) (*Object, *BaseException) {
+	if raised := checkMethodArgs(f, "capitalize", args, StrType); raised != nil {
+		return nil, raised
+	}
+	s := toStrUnsafe(args[0]).Value()
+	buf := bytes.Buffer{}
+	runes := []rune(s)
+	numRunes := len(runes)
+	writeRune := func(r rune, transform func(r rune) rune) {
+		if r <= unicode.MaxASCII {
+			r = transform(r)
+		}
+		buf.WriteRune(r)
+	}
+	if numRunes > 0 {
+		writeRune(runes[0], func(r rune) rune { return unicode.ToUpper(r) })
+	}
+	for i := 1; i < numRunes; i++ {
```

Comment:
I think my current implementation does the right thing for this case:

```
$ cat foo.py 
print '\xd0\xb2\xd0\xbe\xd0\xbb'
print '\xd0\xb2\xd0\xbe\xd0\xbb'.capitalize()
$ ./build/bin/grumpc foo.py > foo.go
$ go run foo.go                     
вол
вол
$ python foo.py 
вол
вол
```

I ran into problems iterating over bytes before because for the multi-byte character case each byte ended up being passed to `buf.WriteRune` separately, which caused issues (presumably b/c the byte was treated as a rune and encoded which in turn caused more bytes to appear in the output).

I could be missing something, though.  My Unicode-fu is a little weak :smile: